data_registry/admin.py

```diff
@@ -132,12 +134,35 @@ def queryset(self, request, queryset):
             return queryset.exclude(qs)
 
 
+class CustomDateFieldListFilter(DateFieldListFilter):
+    def __init__(self, *args, **kwargs):
+        super(CustomDateFieldListFilter, self).__init__(*args, **kwargs)
+
+        now = timezone.now()
+        # When time zone support is enabled, convert "now" to the user's time
+        # zone so Django's definition of "Today" matches what the user expects.
+        if timezone.is_aware(now):
+            now = timezone.localtime(now)
+
+        today = now.date()
+
+        self.links += ((
+            (_('More than a year ago'), {
+                self.lookup_kwarg_until: str(today.replace(year=today.year-1))
+            }),
+            (_('Past year'), {
+                self.lookup_kwarg_since: str(today.year - 1),
+                self.lookup_kwarg_until: str(today.year - 1)
```

Comment:
That code might work, but I think it is clearer below. Also, "past year" can mean "past 365 days" but the query is running "previous year".

```suggestion
            (_('Previous year'), {
                self.lookup_kwarg_since: str(today.replace(year=today.year - 1, month=1, day=1)),
                self.lookup_kwarg_until: str(today.replace(month=1, day=1)),
```